packages/ChangesReloaded-Core.package/CRBrowser.class/instance/delete.st

```diff
@@ -0,0 +1,4 @@
+building
+delete
+
+	SystemChangeNotifier uniqueInstance noMoreNotificationsFor: self
```

Comment:
Note for the in-person review: I want to know what this does :)

Reply:
We have some sliver of an idea but this is why we out-commented it's usage ;)

Reply:
This is a legacy method that we left in for now even though it never gets called because that part is out-commented (see above). We had it in for testing purposes and basically forgot to take it out. Still an interesting research topic ;)